content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,170 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+## Migrating to Cypress 9.3
+
+This guide details the changes and how to change your code to make full use of
+Cypress 9.3.
+[See the full changelog for 9.3](/guides/references/changelog#9-3-0).
+
+None of the changes detailed here are breaking.
```

Comment:
I would get rid of this section and just make the main title 'Migrating to selectFile'. This is similar to the section on cy.route to cy.intercept migration which was in a point release.

Reply:
Ah, that is much nicer. Changing now.